qtpy/QtWidgets.py

```diff
@@ -12,18 +12,27 @@
 
 if PYQT5:
     from PyQt5.QtWidgets import *
+
+    # Fix enums in PyQt5 5.9.*
+    from .enums_compat import demote_enums
+    from PyQt5.QtCore import QT_VERSION_STR as __version__
+    if PYQT5 and __version__.startswith('5.9.'):
+        from PyQt5 import QtWidgets
+        demote_enums(QtWidgets)
+        del QtWidgets
+    del __version__, demote_enums
```

Comment:
```suggestion
    from PyQt5.QtCore import QT_VERSION_STR as __version__
    if __version__.startswith('5.9.'):
        from .enums_compat import demote_enums
        from PyQt5 import QtWidgets
        demote_enums(QtWidgets)
        del QtWidgets, demote_enums
    del __version__
```

* Move `.enums_compt` import and deletion inside the block, so it's only executed if necessary
* Remove redundant `PYQT5` check (since we're already inside the `PYQT5` block)